lms/static/js/views/fields.js

```diff
@@ -562,12 +566,33 @@
                 }
             },
 
+            updateCharCount: function() {
+                var curCharCount,
+                    remainingCharCount,
+                    charCountMessage;
+                // Update character count for textarea
+                if (this.options.maxCharacters) {
+                    curCharCount = $('#u-field-textarea-' + this.options.valueAttribute).val().length;
+                    remainingCharCount = this.options.maxCharacters - curCharCount;
+                    charCountMessage = remainingCharCount === 1 ?
+                        this.options.charCountMessageSingular : this.options.charCountMessage;
+
+                    $('.u-field-footer .cur-char-count').text(remainingCharCount);
```

Comment:
There might be an issue with internationalization by making a sentence by putting two partial sentences together. In some languages the order may be different, where the count needs to be later. My recommendation would be to have a single field with a placeholder in the message for the count.